Package@swift-5.4.swift

```diff
@@ -0,0 +1,73 @@
+// swift-tools-version:5.3
```

Comment:
This copy of Package will need to be deleted.

Reply:
Nope, we have it there so that current consumers are not impacted by the version ramp.